src/js/player.js

```diff
@@ -1292,7 +1292,13 @@ class Player extends Component {
    * @method play
    */
   play() {
-    this.techCall_('play');
+    // Only calls the tech's play if we already have a src loaded
+    if (this.player_.currentSrc()) {
+      this.techCall_('play');
+    } else {
+      this.autoplay(true);
```

Comment:
Another potential issue, on firefox when you hit play without a source, `autplay` is still set to `false`: https://jsfiddle.net/gkatsev/xnm01byt/